.github/workflows/ci.yaml

```diff
@@ -0,0 +1,70 @@
+name: For each commit and PR
+on:
+  push:
+  pull_request:
+
+jobs:
+  validation:
+    runs-on: ubuntu-latest
+    env:
+      CGO_ENABLED: 0
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.14.6'
+    - name: go fmt
+      run: go get golang.org/x/tools/cmd/goimports; goimports -d . | (! grep .)
```

Comment:
this should be `&&`. I think at one point there was a typo or something and the get failed but the command continued because of `;` and goimports failed but because there was no `pipefail` it didn't error and `! grep` worked out fine. Not in this repo I think, maybe another one.

Reply:
Done